### ISSUE TITLE
Bonded keys are not fired together while pressing if they are not a combo;

### ISSUE THREAD

**Issue by se-panfilov** [RobertWHurst/Keystrokes#43]
**Describe the bug**
If you are binding 2 separate keys, e.g.

```js
bindKey('w', () => ... );
bindKey('a', () => ... );
```

While pressing both keys, only one of the callback is triggered. (But both events should be triggered)

**Expected behavior**
I expect both callbacks to be triggered at the same time, cause 2 buttons are still pressed.

**Why**
Imagine a videogame, e.g. a shooter game. User is moving using WASD alongside with "space" for a jump, "G" for throwing grenade and etc. It's literary impossible to bind every combo. Instead the binds should be independent from each other ("W" moves forward, "A" moves left, so eventually user will be moved forward and left) .

**To Reproduce**
An example at stackblitz:
https://stackblitz.com/edit/vitejs-vite-iunkan?file=src%2Fmain.ts

You can move the "Actor" red square using WASD, but cannot move it in a diagonal direction.

**Comment by se-panfilov** [comment]
Is it any chance that this issue will be fixed? =)

**Comment by RobertWHurst** [comment]
Indeed, this is somewhat by design, though I must admit it's not great for games.

For applications, if I have the combo a + b, and a combo a + b + c, I don't really want the first combo firing. The behavior is that keys are consumed by the combos they trigger.

What might make better sense for the gaming use case it the ability to key keystrokes to allow non exclusive combo binds, and also allow overriding this setting when calling bind functions.

I'm quite busy these days, but I'm happy to take a PR until I find the time to do this myself.

Thanks for raising this.
